examples/02-tips-n-tricks/00-example-template.py

```diff
@@ -20,25 +20,18 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-"""
+'''
 .. _ref_how_to_add_an_example_reference_key:
 
+============================
 Adding a New Gallery Example
-----------------------------
+============================
 
-This example demonstrates how to add new examples as well as being a template that can
-be used in their creation.
+This example demonstrates how to add new examples as well as being a template
+that can be used in their creation.
```

Comment:
```suggestion
This example demonstrates how to add new examples as well as being a template
that can be used in their creation.
```
 I don't understand the above sentence. Are you saying:
 This example demonstrates how to add new examples, which can be used as templates
for the creation of other examples.